### DISCUSSION TITLE
Where Is the UI for configuration.

### DISCUSSION THREAD

**Discussion by CavemanTitan** [mjmeli/ha-duke-energy-gateway#89] [Q&A] [answered]
I added the duke energy gateway to the HACS integrations. Ive rebooted the host. I am having trouble finding where the UI would be to enter in my email, password, and to select the meter.  In integrations, I can click on Duke Energy Gateway, and get the help file, or click on the 3 dots, and get options for - information, repository, update information, redownload, open issue.   

Any assistance would be appreciated.  Thank you.

**Comment by mjmeli** [comment, marked as answer]
It sounds like you might be looking in the HACS sub-menu. You need to go to the Home Assistant integrations page (Configuration > Devices & Services) then it will be under the "Add Integration" button. If it's still not there, you might need to do a hard refresh and clear browser cache which is Ctrl + F5 on most browsers.
